src/Cake.Web/App_Data/docs/fundamentals/preprocessor-directives.md

```diff
@@ -21,15 +21,34 @@ Right now, only NuGet packages are supported.
 
 ### Load directive
 The load directive is used to reference external Cake scripts. Useful i.e. if you have common utility functions.
+Starting from 0.18.0 you can also load cake scripts from nuget.
 
 #### Usage
-The directive has one parameter which is the location to the script.
+The directive has one parameter which is the location to the script which optionally includes a scheme: `local` or `nuget`. The default is `local`. 
 
+###### Default:
 ```csharp
 #l "utilities.cake"
```

Comment:
Perhaps should use `scripts/utilities.cake` like the `Local scheme` example below?